docs/proto/index.html

```diff
@@ -2976,14 +2976,13 @@ <h3 id="username.RegisterTokenMsg">RegisterTokenMsg</h3>
                   <td>targets</td>
                   <td><a href="#username.BlockchainAddress">BlockchainAddress</a></td>
                   <td>repeated</td>
-                  <td><p>Targets is a blockchain address list that this token should point to. </p></td>
-                </tr>
-              
-                <tr>
-                  <td>owner</td>
-                  <td><a href="#bytes">bytes</a></td>
-                  <td></td>
-                  <td><p>Owner is the address that the newly created token will belong to. </p></td>
+                  <td><p>Targets is a blockchain address list that this token should point to.
+
+This change was reverted because we do not want to support additional
+functionality in a deprecated module.
+
+Owner is the address that the newly created token will belong to.
+bytes owner = 4 [(gogoproto.casttype) = &#34;github.com/iov-one/weave.Address&#34;]; </p></td>
```

Comment:
Strange docs generation. The comments below `targets` are added to `targets` docs. Can you move the comments out of `RegisterTokenMsg` or remove them entirely?

Reply:
Done in https://github.com/iov-one/weave/pull/1079/commits/ed68e0a36d7419d843bcec02929179a0c8a3d39c